dipy/reconst/ivim.py

```diff
@@ -29,7 +29,7 @@ def ivim_function(params, bvals):
 def _ivim_error(params, bvals, signal):
     """Error function to be used in fitting the model
     """
-    return np.linalg.norm(signal - ivim_function(params, bvals))
+    return (signal - ivim_function(params, bvals))**2
```

Comment:
If you are using `leastsq`, you want to return the the error (not the squared error):
    
     (signal - ivim_function(params, bvals))